commune_app/User.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
```

Comment:
It looks like you have this file twice:
 - 1 here - commune_app/User.py
 - 1 in commune_app/models/User.py
Please remove this file.